### PR TITLE
Revert to previous minimum glibc version of ≥2.17 instead of ≥2.28

### PR DESCRIPTION
**Pull request by tsibley**
Resolves: <https://github.com/nextstrain/conda-base/issues/92>

## Checklist

- [x] Checks pass

### CODE REVIEW

File: src/recipe.yaml [hunk 1]
```diff
@@ -87,3 +87,15 @@ requirements:
     - xz
     - zip
     - zstd
+
+    # Avoid a newer minimum glibc version introduced by c-ares 1.33.0¹.  This
+    # pin keeps the previous glibc requirement of ≥2.17 instead of ≥2.28.²
+    # Eventually this pin will become unworkable, but we can keep it for now.
```
Comment (victorlin) [inline src/recipe.yaml:93]: Is the idea that eventually other package pins will fail to resolve with `c-ares <1.33.0` during the locking step or could something happen during runtime?

**Review by victorlin — APPROVED** [review]
Setup successfully on macOS 13.6

<details>
<summary>
output snippet
</summary>

```
Setting up conda…
Removing existing directory /Users/vlin/.nextstrain/runtimes/conda/micromamba to start fresh…
Requesting Micromamba from https://api.anaconda.org/download/conda-forge/micromamba/1.1.0/osx-64/micromamba-1.1.0-1.tar.bz2…
Downloading and extracting Micromamba to /Users/vlin/.nextstrain/runtimes/conda/micromamba…
Removing existing directory /Users/vlin/.nextstrain/runtimes/conda/env to start fresh…
Installing Conda packages into /Users/vlin/.nextstrain/runtimes/conda/env…
  - nextstrain-base ==20240916T182550Z

                                           __
          __  ______ ___  ____ _____ ___  / /_  ____ _
         / / / / __ `__ \/ __ `/ __ `__ \/ __ \/ __ `/
        / /_/ / / / / / / /_/ / / / / / / /_/ / /_/ /
       / .___/_/ /_/ /_/\__,_/_/ /_/ /_/_.___/\__,_/
      /_/

nextstrain/label/pull-93/osx-64                      4.6kB @   9.9kB/s  0.5s
nextstrain/label/pull-93/noarch                    130.0 B @ 239.0 B/s  0.5s
bioconda/osx-64                                      4.6MB @   6.0MB/s  0.8s
bioconda/noarch                                      5.2MB @   4.4MB/s  0.8s
conda-forge/noarch                                  19.0MB @   7.7MB/s  2.8s
conda-forge/osx-64                                  39.3MB @   6.9MB/s  6.4s

Transaction

  Prefix: /Users/vlin/.nextstrain/runtimes/conda/env

  Updating specs:

   - nextstrain-base==20240916T182550Z


  Package                              Version  Build                  Channel                              Size
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────
  Install:
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────

  + absl-py                              2.1.0  pyhd8ed1ab_0           conda-forge/noarch                  107kB
  + aiobotocore                         2.15.0  pyhd8ed1ab_0           conda-forge/noarch                   67kB
  + aioeasywebdav                        2.4.0  pyha770c72_0           conda-forge/noarch                   13kB
  + aiohappyeyeballs                     2.4.0  pyhd8ed1ab_0           conda-forge/noarch                   17kB
  + aiohttp                             3.10.5  py311h3336109_1        conda-forge/osx-64                  797kB
  + aioitertools                        0.11.0  pyhd8ed1ab_0           conda-forge/noarch                   23kB
  + aiosignal                            1.3.1  pyhd8ed1ab_0           conda-forge/noarch                   13kB
  + amply                                0.1.6  pyhd8ed1ab_0           conda-forge/noarch                   22kB
  + appdirs                              1.4.4  pyh9f0ad1d_0           conda-forge/noarch                   13kB
  + attmap                              0.13.2  pyhd8ed1ab_0           conda-forge/noarch                   14kB
  + attrs                               24.2.0  pyh71513ae_0           conda-forge/noarch                   56kB
  + augur                               25.4.0  pyhdfd78af_0           bioconda/noarch                     260kB
  + auspice                             2.58.0  h4e39fe6_0             bioconda/osx-64                      40MB
  + aws-c-auth                          0.7.27  h77ec9d9_0             conda-forge/osx-64                   94kB
  + aws-c-cal                            0.7.4  h3e75f19_0             conda-forge/osx-64                   39kB
  + aws-c-common                        0.9.27  hfdf4475_0             conda-forge/osx-64                  225kB
  + aws-c-compression                   0.2.19  h3e75f19_0             conda-forge/osx-64                   18kB
  + aws-c-event-stream                   0.4.3  h324d61a_0             conda-forge/osx-64                   46kB
  + aws-c-http                           0.8.8  h504e0bf_1             conda-forge/osx-64                  164kB
  + aws-c-io                           0.14.18  hef79b51_8             conda-forge/osx-64                  139kB
  + aws-c-mqtt                          0.10.4  h03607b6_18            conda-forge/osx-64                  138kB
  + aws-c-s3                             0.6.5  h74e0911_0             conda-forge/osx-64                   97kB
  + aws-c-sdkutils                      0.1.19  hf37c103_2             conda-forge/osx-64                   50kB
  + aws-checksums                       0.1.18  hf37c103_10            conda-forge/osx-64                   49kB
  + awscli                             2.17.50  py311h6eed73b_0        conda-forge/osx-64                   13MB
  + awscrt                              0.21.2  py311hf5f6ca4_18       conda-forge/osx-64                  184kB
  + bash                                5.2.21  h0f89e9d_0             conda-forge/osx-64                    2MB
  + bcbio-gff                            0.7.1  pyh7e72e81_2           bioconda/noarch                      22kB
  + bcrypt                               4.2.0  py311h95688db_1        conda-forge/osx-64                  222kB
  + biopython                             1.84  py311h72ae277_0        conda-forge/osx-64                    3MB
  + blackjax                             0.9.6  pyhd8ed1ab_0           conda-forge/noarch                   80kB
  + boto3                              1.35.16  pyhd8ed1ab_0           conda-forge/noarch                   81kB
  + botocore                           1.35.16  pyge310_1234567_0      conda-forge/noarch                    7MB
  + bracex                               2.2.1  pyhd8ed1ab_0           conda-forge/noarch                   14kB
  + brotli                               1.1.0  h00291cd_2             conda-forge/osx-64                   19kB
  + brotli-bin                           1.1.0  h00291cd_2             conda-forge/osx-64                   17kB
  + brotli-python                        1.1.0  py311hd89902b_2        conda-forge/osx-64                  364kB
  + bx-python                           0.11.0  py311h2145810_3        bioconda/osx-64                       1MB
  + bzip2                                1.0.8  hfdf4475_7             conda-forge/osx-64                  134kB
  + c-ares                              1.32.3  h51dda26_0             conda-forge/osx-64                  160kB
  + ca-certificates                  2024.8.30  h8857fd0_0             conda-forge/osx-64                  159kB
  + cachetools                           5.5.0  pyhd8ed1ab_0           conda-forge/noarch                   15kB
  + certifi                          2024.8.30  pyhd8ed1ab_0           conda-forge/noarch                  164kB
  + cffi                                1.17.1  py311h137bacd_0        conda-forge/osx-64                  289kB
  + charset-normalizer                   3.3.2  pyhd8ed1ab_0           conda-forge/noarch                   47kB
  + coin-or-cbc                        2.10.12  h26cd4a2_1             conda-forge/osx-64                  894kB
  + coin-or-cgl                         0.60.9  h5c63661_0             conda-forge/osx-64                  529kB
  + coin-or-clp                        1.17.10  hf05e7e8_0             conda-forge/osx-64                    1MB
  + coin-or-osi                       0.108.11  h8c189bd_1             conda-forge/osx-64                  343kB
  + coin-or-utils                      2.11.12  h95c8798_1             conda-forge/osx-64                  630kB
  + coincbc                            2.10.12  1_metapackage          conda-forge/noarch                   12kB
  + colorama                             0.4.6  pyhd8ed1ab_0           conda-forge/noarch                   25kB
  + configargparse                         1.7  pyhd8ed1ab_0           conda-forge/noarch                   39kB
  + connection_pool                      0.0.3  pyhd3deb0d_0           conda-forge/noarch                    8kB
  + contourpy                            1.3.0  py311hf2f7c97_1        conda-forge/osx-64                  258kB
  + coreutils                              9.5  h10d778d_0             conda-forge/osx-64                    1MB
  + cryptography                        40.0.2  py311h61927ef_0        conda-forge/osx-64                    1MB
  + csvtk                               0.30.0  h527b516_2             bioconda/osx-64                       9MB
  + curl                                8.10.0  h58e7537_0             conda-forge/osx-64                  160kB
  + cvxopt                               1.3.2  py311h2135409_3        conda-forge/osx-64                  524kB
  + cycler                              0.12.1  pyhd8ed1ab_0           conda-forge/noarch                   13kB
  + datrie                               0.8.2  py311h3336109_8        conda-forge/osx-64                  135kB
  + defusedxml                           0.7.1  pyhd8ed1ab_0           conda-forge/noarch                   24kB
  + distro                               1.8.0  pyhd8ed1ab_0           conda-forge/noarch                   41kB
  + docutils                              0.19  py311h6eed73b_1        conda-forge/osx-64                  973kB
  + dpath                                2.2.0  pyha770c72_0           conda-forge/noarch                   21kB
  + dropbox                             12.0.2  pyhd8ed1ab_0           conda-forge/noarch                  966kB
  + dsdp                                   5.8  h6e329d1_1203          conda-forge/osx-64                  226kB
  + eido                                 0.2.2  pyhd8ed1ab_0           conda-forge/noarch                   20kB
  + epiweeks                             2.3.0  pyhdfd78af_0           bioconda/noarch                      14kB
  + evofr                               0.1.23  pyhdfd78af_0           bioconda/noarch                      65kB
  + exceptiongroup                       1.2.2  pyhd8ed1ab_0           conda-forge/noarch                   20kB
  + fasteners                           0.17.3  pyhd8ed1ab_0           conda-forge/noarch                   20kB
  + fastprogress                         1.0.3  pyhd8ed1ab_0           conda-forge/noarch                   17kB
  + fasttree                            2.1.11  h26a06ae_4             bioconda/osx-64                     238kB
  + fftw                                3.3.10  nompi_h292e606_110     conda-forge/osx-64                    2MB
  + filechunkio                            1.8  py_2                   conda-forge/noarch                    5kB
  + fonttools                           4.53.1  py311h3336109_1        conda-forge/osx-64                    3MB
  + freetype                            2.12.1  h60636b9_2             conda-forge/osx-64                  599kB
  + frozenlist                           1.4.1  py311h3336109_1        conda-forge/osx-64                   53kB
  + fsspec                            2024.9.0  pyhff2d567_0           conda-forge/noarch                  134kB
  + ftputil                              5.1.0  pyhd8ed1ab_0           conda-forge/noarch                   47kB
  + gawk                                 5.3.0  h2c496e9_0             conda-forge/osx-64                    1MB
  + gdbm                                  1.18  h8a0c380_2             conda-forge/osx-64                  134kB
  + gettext                             0.22.5  hdfe23c8_3             conda-forge/osx-64                  480kB
  + gettext-tools                       0.22.5  hdfe23c8_3             conda-forge/osx-64                    3MB
  + git                                 2.46.0  pl5321h9b6aa9b_0       conda-forge/osx-64                   11MB
  + gitdb                               4.0.11  pyhd8ed1ab_0           conda-forge/noarch                   53kB
  + gitpython                           3.1.43  pyhd8ed1ab_0           conda-forge/noarch                  157kB
  + glpk                                   5.0  h3cb5acd_0             conda-forge/osx-64                    1MB
  + gmp                                  6.3.0  hf036a51_2             conda-forge/osx-64                  429kB
  + google-api-core                     2.19.2  pyhd8ed1ab_0           conda-forge/noarch                   85kB
  + google-api-python-client           2.145.0  pyhd8ed1ab_0           conda-forge/noarch                    6MB
  + google-auth                         2.34.0  pyhff2d567_0           conda-forge/noarch                  112kB
  + google-auth-httplib2                 0.2.0  pyhd8ed1ab_0           conda-forge/noarch                   15kB
  + google-cloud-core                    2.4.1  pyhd8ed1ab_0           conda-forge/noarch                   28kB
  + google-cloud-storage                2.18.2  pyhff2d567_0           conda-forge/noarch                   91kB
  + google-crc32c                        1.1.2  py311h0b57020_5        conda-forge/osx-64                   24kB
  + google-resumable-media               2.7.2  pyhd8ed1ab_1           conda-forge/noarch                   46kB
  + googleapis-common-protos            1.65.0  pyhd8ed1ab_0           conda-forge/noarch                  116kB
  + grpcio                              1.62.2  py311h78ff076_0        conda-forge/osx-64                  988kB
  + gsl                                    2.7  h93259b0_0             conda-forge/osx-64                    3MB
  + gzip                                  1.13  h37ebe6b_0             conda-forge/osx-64                  113kB
  + hdbscan                       0.8.38.post1  py311h0034819_2        conda-forge/osx-64                  559kB
  + httplib2                            0.22.0  pyhd8ed1ab_0           conda-forge/noarch                   95kB
  + humanfriendly                         10.0  pyhd8ed1ab_6           conda-forge/noarch                   73kB
  + icu                                   73.2  hf5e326d_0             conda-forge/osx-64                   12MB
  + idna                                  3.10  pyhd8ed1ab_0           conda-forge/noarch                   50kB
  + importlib-metadata                   8.5.0  pyha770c72_0           conda-forge/noarch                   29kB
  + iniconfig                            2.0.0  pyhd8ed1ab_0           conda-forge/noarch                   11kB
  + iqtree                               2.3.6  h49642b6_0             bioconda/osx-64                       4MB
  + isa-l                               2.31.0  hfdf4475_2             conda-forge/osx-64                  150kB
  + isodate                              0.6.1  pyhd8ed1ab_0           conda-forge/noarch                   29kB
  + jax                                 0.4.31  pyhd8ed1ab_1           conda-forge/noarch                    1MB
  + jaxlib                              0.4.31  cpu_py311h88acc01_1    conda-forge/osx-64                   58MB
  + jaxopt                               0.8.3  pyhd8ed1ab_0           conda-forge/noarch                  103kB
  + jinja2                               3.1.4  pyhd8ed1ab_0           conda-forge/noarch                  112kB
  + jmespath                             1.0.1  pyhd8ed1ab_0           conda-forge/noarch                   21kB
  + joblib                               1.4.2  pyhd8ed1ab_0           conda-forge/noarch                  220kB
  + jq                                   1.7.1  h10d778d_0             conda-forge/osx-64                  304kB
  + jsonschema                           3.2.0  pyhd8ed1ab_3           conda-forge/noarch                   46kB
  + jupyter_core                         5.7.2  py311h6eed73b_0        conda-forge/osx-64                   96kB
  + kiwisolver                           1.4.7  py311hf2f7c97_0        conda-forge/osx-64                   60kB
  + krb5                                1.21.3  h37d8d59_0             conda-forge/osx-64                    1MB
  + lcms2                                 2.16  ha2f27b4_0             conda-forge/osx-64                  224kB
  + lerc                                 4.0.0  hb486fe8_0             conda-forge/osx-64                  290kB
  + libabseil                       20240116.2  cxx17_hf036a51_1       conda-forge/osx-64                    1MB
  + libasprintf                         0.22.5  hdfe23c8_3             conda-forge/osx-64                   40kB
  + libasprintf-devel                   0.22.5  hdfe23c8_3             conda-forge/osx-64                   35kB
  + libblas                              3.9.0  22_osx64_openblas      conda-forge/osx-64                   15kB
  + libbrotlicommon                      1.1.0  h00291cd_2             conda-forge/osx-64                   67kB
  + libbrotlidec                         1.1.0  h00291cd_2             conda-forge/osx-64                   30kB
  + libbrotlienc                         1.1.0  h00291cd_2             conda-forge/osx-64                  296kB
  + libcblas                             3.9.0  22_osx64_openblas      conda-forge/osx-64                   15kB
  + libcrc32c                            1.1.2  he49afe7_0             conda-forge/osx-64                   20kB
  + libcurl                             8.10.0  h58e7537_0             conda-forge/osx-64                  402kB
  + libcxx                              18.1.8  hd876a4e_7             conda-forge/osx-64                  439kB
  + libdeflate                            1.21  hfdf4475_0             conda-forge/osx-64                   71kB
  + libedit                       3.1.20191231  h0678c8f_2             conda-forge/osx-64                  105kB
  + libev                                 4.33  h10d778d_2             conda-forge/osx-64                  107kB
  + libexpat                             2.6.3  hac325c4_0             conda-forge/osx-64                   71kB
  + libffi                               3.4.2  h0d85af4_5             conda-forge/osx-64                   51kB
  + libgettextpo                        0.22.5  hdfe23c8_3             conda-forge/osx-64                  172kB
  + libgettextpo-devel                  0.22.5  hdfe23c8_3             conda-forge/osx-64                   37kB
  + libgfortran                          5.0.0  13_2_0_h97931a8_3      conda-forge/osx-64                  110kB
  + libgfortran5                        13.2.0  h2873a65_3             conda-forge/osx-64                    2MB
  + libgrpc                             1.62.2  h384b2fc_0             conda-forge/osx-64                    5MB
  + libhwloc                            2.11.1  default_h456cccd_1000  conda-forge/osx-64                    2MB
  + libiconv                              1.17  hd75f5a5_2             conda-forge/osx-64                  667kB
  + libidn2                              2.3.7  h10d778d_0             conda-forge/osx-64                  133kB
  + libintl                             0.22.5  hdfe23c8_3             conda-forge/osx-64                   88kB
  + libintl-devel                       0.22.5  hdfe23c8_3             conda-forge/osx-64                   38kB
  + libjpeg-turbo                        3.0.0  h0dc2134_1             conda-forge/osx-64                  580kB
  + liblapack                            3.9.0  22_osx64_openblas      conda-forge/osx-64                   15kB
  + liblapacke                           3.9.0  22_osx64_openblas      conda-forge/osx-64                   15kB
  + libllvm14                           14.0.6  hc8e404f_4             conda-forge/osx-64                   22MB
  + libnghttp2                          1.58.0  h64cf6d3_1             conda-forge/osx-64                  600kB
  + libopenblas                         0.3.27  openmp_h8869122_1      conda-forge/osx-64                    6MB
  + libpng                              1.6.44  h4b8f8c9_0             conda-forge/osx-64                  268kB
  + libprotobuf                         4.25.3  h4e4d658_0             conda-forge/osx-64                    2MB
  + libre2-11                       2023.09.01  h81f5012_2             conda-forge/osx-64                  184kB
  + libsodium                           1.0.20  hfdf4475_0             conda-forge/osx-64                  210kB
  + libsqlite                           3.46.1  h4b8f8c9_0             conda-forge/osx-64                  908kB
  + libssh2                             1.11.0  hd019ec5_0             conda-forge/osx-64                  260kB
  + libtiff                              4.6.0  h603087a_4             conda-forge/osx-64                  258kB
  + libunistring                        0.9.10  h0d85af4_0             conda-forge/osx-64                    1MB
  + libuv                               1.48.0  h67532ce_0             conda-forge/osx-64                  407kB
  + libwebp-base                         1.4.0  h10d778d_0             conda-forge/osx-64                  355kB
  + libxcb                                1.16  h00291cd_1             conda-forge/osx-64                  324kB
  + libxml2                             2.12.7  hc603aa4_3             conda-forge/osx-64                  620kB
  + libzlib                              1.3.1  h87427d6_1             conda-forge/osx-64                   57kB
  + llvm-openmp                         18.1.8  h15ab845_1             conda-forge/osx-64                  300kB
  + llvmlite                            0.41.1  py311hb5c2e0a_0        conda-forge/osx-64                  367kB
  + logmuse                              0.2.6  pyh8c360ce_0           conda-forge/noarch                   11kB
  + mafft                                7.526  hfdf4475_0             conda-forge/osx-64                    5MB
  + markdown-it-py                       3.0.0  pyhd8ed1ab_0           conda-forge/noarch                   64kB
  + markupsafe                           2.1.5  py311h3336109_1        conda-forge/osx-64                   26kB
  + matplotlib-base                      3.9.2  py311h8b21175_1        conda-forge/osx-64                    8MB
  + mdurl                                0.1.2  pyhd8ed1ab_0           conda-forge/noarch                   15kB
  + metis                                5.1.0  he965462_1007          conda-forge/osx-64                    4MB
  + ml_dtypes                            0.5.0  py311haeb46be_0        conda-forge/osx-64                  220kB
  + mpfr                                 4.2.1  haed47dc_3             conda-forge/osx-64                  373kB
  + multidict                            6.1.0  py311h3e662af_0        conda-forge/osx-64                   56kB
  + multipledispatch                     0.6.0  pyhd8ed1ab_1           conda-forge/noarch                   17kB
  + munkres                              1.1.4  pyh9f0ad1d_0           conda-forge/noarch                   12kB
  + nbformat                            5.10.4  pyhd8ed1ab_0           conda-forge/noarch                  101kB
  + ncbi-datasets-cli                  16.28.0  h694c41f_0             conda-forge/osx-64                   33MB
  + ncurses                                6.5  hf036a51_1             conda-forge/osx-64                  822kB
  + networkx                               3.3  pyhd8ed1ab_1           conda-forge/noarch                    1MB
  + nextalign                           2.14.0  h527b516_2             bioconda/osx-64                       2MB
  + nextclade                            3.8.2  h527b516_0             bioconda/osx-64                       4MB
  + nextclade2                          2.14.0  h527b516_2             bioconda/osx-64                       4MB
  + nextstrain-base           20240916T182550Z  h0dc7051_0_locked      nextstrain/label/pull-93/osx-64      12kB
  + nextstrain-cli                       8.5.3  pyhdfd78af_0           bioconda/noarch                     130kB
  + nodejs                             20.12.2  hfc0f20e_0             conda-forge/osx-64                   12MB
  + numba                               0.58.1  py311h97119f7_0        conda-forge/osx-64                    6MB
  + numpy                               1.26.4  py311hc43a94b_0        conda-forge/osx-64                    8MB
  + numpyro                             0.13.2  pyhd8ed1ab_1           conda-forge/noarch                  229kB
  + oauth2client                         4.1.3  py_0                   conda-forge/noarch                   67kB
  + oniguruma                            6.9.9  h10d778d_0             conda-forge/osx-64                  224kB
  + openjpeg                             2.5.2  h7310d3a_0             conda-forge/osx-64                  331kB
  + openssl                              3.3.2  hd23fc13_0             conda-forge/osx-64                    3MB
  + opt-einsum                           3.3.0  hd8ed1ab_2             conda-forge/noarch                    7kB
  + opt_einsum                           3.3.0  pyhc1e730c_2           conda-forge/noarch                   58kB
  + packaging                             24.1  pyhd8ed1ab_0           conda-forge/noarch                   50kB
  + pandas                               1.5.3  py311hd84f3f5_1        conda-forge/osx-64                   14MB
  + pango_aliasor                        0.3.0  pyhdfd78af_0           bioconda/noarch                      11kB
  + paramiko                             3.4.1  pyhd8ed1ab_0           conda-forge/noarch                  160kB
  + pathogen-embed                       3.1.0  pyhdfd78af_0           bioconda/noarch                      25kB
  + pbzip2                              1.1.13  h92b232e_2             conda-forge/osx-64                   41kB
  + pcre2                                10.44  h7634a1b_2             conda-forge/osx-64                  854kB
  + peppy                               0.40.6  pyhd8ed1ab_0           conda-forge/noarch                   29kB
  + perl                                5.32.1  7_h10d778d_perl5       conda-forge/osx-64                   12MB
  + pigz                                   2.8  h92b6c6a_0             conda-forge/osx-64                   70kB
  + pillow                              10.4.0  py311h17ad1af_1        conda-forge/osx-64                   42MB
  + pip                                   24.2  pyh8b19718_1           conda-forge/noarch                    1MB
  + plac                                 1.4.3  pyhd8ed1ab_0           conda-forge/noarch                   26kB
  + platformdirs                         4.3.3  pyhd8ed1ab_0           conda-forge/noarch                   21kB
  + pluggy                               1.5.0  pyhd8ed1ab_0           conda-forge/noarch                   24kB
  + ply                                   3.11  pyhd8ed1ab_2           conda-forge/noarch                   49kB
  + prettytable                         3.11.0  pyhd8ed1ab_0           conda-forge/noarch                   30kB
  + prompt-toolkit                      3.0.38  pyha770c72_0           conda-forge/noarch                  269kB
  + prompt_toolkit                      3.0.38  hd8ed1ab_0             conda-forge/noarch                    6kB
  + proto-plus                          1.23.0  pyhd8ed1ab_0           conda-forge/noarch                   42kB
  + protobuf                            4.25.3  py311h6b31176_1        conda-forge/osx-64                  381kB
  + psutil                               6.0.0  py311h3336109_1        conda-forge/osx-64                  515kB
  + pthread-stubs                          0.4  hc929b4f_1001          conda-forge/osx-64                    6kB
  + pulp                                 2.7.0  py311h6eed73b_1        conda-forge/osx-64                  198kB
  + pyasn1                               0.6.1  pyhd8ed1ab_0           conda-forge/noarch                   62kB
  + pyasn1-modules                       0.4.1  pyhd8ed1ab_0           conda-forge/noarch                   96kB
  + pycparser                             2.22  pyhd8ed1ab_0           conda-forge/noarch                  105kB
  + pyfastx                              2.1.0  py311h45763dd_3        bioconda/osx-64                     659kB
  + pygments                            2.18.0  pyhd8ed1ab_0           conda-forge/noarch                  879kB
  + pyjwt                                2.9.0  pyhd8ed1ab_1           conda-forge/noarch                   24kB
  + pynacl                               1.5.0  py311h3336109_4        conda-forge/osx-64                    1MB
  + pynndescent                         0.5.13  pyhff2d567_0           conda-forge/noarch                   50kB
  + pyopenssl                           23.1.1  pyhd8ed1ab_0           conda-forge/noarch                  128kB
  + pyparsing                            3.1.4  pyhd8ed1ab_0           conda-forge/noarch                   90kB
  + pyrsistent                          0.20.0  py311h3336109_1        conda-forge/osx-64                  123kB
  + pysftp                               0.2.9  py_1                   conda-forge/noarch                   16kB
  + pysocks                              1.7.1  pyha2e5f31_6           conda-forge/noarch                   19kB
  + pytest                               8.3.3  pyhd8ed1ab_0           conda-forge/noarch                  258kB
  + python                             3.11.10  h8f8b54e_0_cpython     conda-forge/osx-64                   15MB
  + python-calamine                      0.2.3  py311h95688db_1        conda-forge/osx-64                  658kB
  + python-dateutil                      2.8.2  pyhd8ed1ab_0           conda-forge/noarch                  246kB
  + python-fastjsonschema               2.20.0  pyhd8ed1ab_0           conda-forge/noarch                  226kB
  + python-irodsclient                   2.1.0  pyhd8ed1ab_0           conda-forge/noarch                  186kB
  + python-isal                          1.7.0  py311h3336109_1        conda-forge/osx-64                   63kB
  + python-zlib-ng                       0.5.0  py311h121a69e_1        conda-forge/osx-64                   55kB
  + python_abi                            3.11  5_cp311                conda-forge/osx-64                    6kB
  + pytz                                2024.2  pyhd8ed1ab_0           conda-forge/noarch                  187kB
  + pyu2f                                0.1.5  pyhd8ed1ab_0           conda-forge/noarch                   32kB
  + pyyaml                               6.0.2  py311h3336109_1        conda-forge/osx-64                  194kB
  + qhull                               2020.2  h3c5361c_5             conda-forge/osx-64                  528kB
  + raxml                               8.2.13  h8af04d4_2             bioconda/osx-64                       3MB
  + re2                             2023.09.01  hb168e87_2             conda-forge/osx-64                   27kB
  + readline                               8.2  h9e318b2_1             conda-forge/osx-64                  256kB
  + requests                            2.32.3  pyhd8ed1ab_0           conda-forge/noarch                   59kB
  + reretry                             0.11.8  pyhd8ed1ab_0           conda-forge/noarch                   12kB
  + rich                                13.8.1  pyhd8ed1ab_0           conda-forge/noarch                  185kB
  + rsa                                    4.9  pyhd8ed1ab_0           conda-forge/noarch                   30kB
  + ruamel.yaml                        0.17.21  py311h5547dcb_3        conda-forge/osx-64                  261kB
  + ruamel.yaml.clib                     0.2.7  py311h2725bcf_2        conda-forge/osx-64                  117kB
  + ruby                                 3.3.3  ha604482_0             conda-forge/osx-64                    9MB
  + s3fs                              2024.9.0  pyhd8ed1ab_0           conda-forge/noarch                   32kB
  + s3transfer                          0.10.2  pyhd8ed1ab_0           conda-forge/noarch                   63kB
  + scikit-learn                         1.4.2  py311h3c3ac6d_1        conda-forge/osx-64                    9MB
  + scipy                               1.14.1  py311hb3ed397_0        conda-forge/osx-64                   16MB
  + seqkit                               2.8.2  h527b516_1             bioconda/osx-64                       8MB
  + setuptools                          73.0.1  pyhd8ed1ab_0           conda-forge/noarch                    1MB
  + setuptools-scm                       8.1.0  pyhd8ed1ab_0           conda-forge/noarch                   38kB
  + six                                 1.16.0  pyh6c4a22f_0           conda-forge/noarch                   14kB
  + slacker                             0.14.0  py_0                   conda-forge/noarch                   16kB
  + smart_open                           7.0.4  pyhd8ed1ab_0           conda-forge/noarch                   52kB
  + smmap                                5.0.0  pyhd8ed1ab_0           conda-forge/noarch                   22kB
  + snakemake                           7.32.4  hdfd78af_1             bioconda/noarch                       9kB
  + snakemake-minimal                   7.32.4  pyhdfd78af_1           bioconda/noarch                     278kB
  + sqlite                              3.46.1  he26b093_0             conda-forge/osx-64                  912kB
  + stone                                3.3.2  pyhd8ed1ab_0           conda-forge/noarch                  119kB
  + stopit                               1.1.2  py_0                   conda-forge/noarch                   16kB
  + suitesparse                          7.8.2  ha93ada7_0             conda-forge/osx-64                    2MB
  + tabulate                             0.9.0  pyhd8ed1ab_1           conda-forge/noarch                   36kB
  + tbb                              2021.13.0  h37c8870_0             conda-forge/osx-64                  159kB
  + threadpoolctl                        3.5.0  pyhc1e730c_0           conda-forge/noarch                   24kB
  + throttler                            1.2.2  pyhd8ed1ab_0           conda-forge/noarch                   12kB
  + tk                                  8.6.13  h1abcd95_1             conda-forge/osx-64                    3MB
  + tomli                                2.0.1  pyhd8ed1ab_0           conda-forge/noarch                   16kB
  + toposort                              1.10  pyhd8ed1ab_0           conda-forge/noarch                   14kB
  + tqdm                                4.66.5  pyhd8ed1ab_0           conda-forge/noarch                   90kB
  + traitlets                           5.14.3  pyhd8ed1ab_0           conda-forge/noarch                  110kB
  + treetime                            0.11.4  pyhdfd78af_0           bioconda/noarch                     122kB
  + tsv-utils                            2.2.3  hfdf4475_7             conda-forge/osx-64                    2MB
  + typing-extensions                   4.12.2  hd8ed1ab_0             conda-forge/noarch                   10kB
  + typing_extensions                   4.12.2  pyha770c72_0           conda-forge/noarch                   40kB
  + tzdata                               2024a  h8827d51_1             conda-forge/noarch                  124kB
  + ubiquerg                             0.8.0  pyhd8ed1ab_0           conda-forge/noarch                   21kB
  + umap-learn                           0.5.6  py311h6eed73b_1        conda-forge/osx-64                  189kB
  + unzip                                  6.0  h0d85af4_3             conda-forge/osx-64                  170kB
  + uritemplate                          4.1.1  pyhd8ed1ab_0           conda-forge/noarch                   12kB
  + urllib3                            1.26.19  pyhd8ed1ab_0           conda-forge/noarch                  115kB
  + vcftools                            0.1.16  pl5321h447d7a5_11      bioconda/osx-64                     483kB
  + veracitools                          0.1.3  py_0                   conda-forge/noarch                    6kB
  + wcmatch                                9.0  pyhd8ed1ab_0           conda-forge/noarch                   37kB
  + wcwidth                             0.2.13  pyhd8ed1ab_0           conda-forge/noarch                   33kB
  + wget                                1.21.4  hca547e6_0             conda-forge/osx-64                  778kB
  + wheel                               0.44.0  pyhd8ed1ab_0           conda-forge/noarch                   59kB
  + wrapt                               1.16.0  py311h3336109_1        conda-forge/osx-64                   60kB
  + xopen                                2.0.2  pyh707e725_1           conda-forge/noarch                   22kB
  + xorg-libxau                         1.0.11  h0dc2134_0             conda-forge/osx-64                   13kB
  + xorg-libxdmcp                        1.1.3  h35c211d_0             conda-forge/osx-64                   17kB
  + xz                                   5.2.6  h775f41a_0             conda-forge/osx-64                  238kB
  + yaml                                 0.2.5  h0d85af4_2             conda-forge/osx-64                   84kB
  + yarl                                 1.9.4  py311h3336109_1        conda-forge/osx-64                  113kB
  + yte                                  1.5.4  pyha770c72_0           conda-forge/noarch                   14kB
  + zip                                    3.0  h0dc2134_3             conda-forge/osx-64                  173kB
  + zipp                                3.20.2  pyhd8ed1ab_0           conda-forge/noarch                   21kB
  + zlib                                 1.3.1  h87427d6_1             conda-forge/osx-64                   89kB
  + zlib-ng                              2.2.1  hf036a51_0             conda-forge/osx-64                  104kB
  + zstandard                           0.23.0  py311hdf6fcd6_1        conda-forge/osx-64                  411kB
  + zstd                                 1.5.6  h915ae27_0             conda-forge/osx-64                  499kB

  Summary:

  Install: 320 packages

  Total download: 498MB
```

</details>

File: src/recipe.yaml (hunk 1 above)
Comment (tsibley) [reply src/recipe.yaml:93]: It should happen during the locking step (i.e. package build time before we release/publish).

**Comment by tsibley** [comment]
An alternative implementation if pinning `c-ares` becomes problematic for some technical/implementation reason (i.e. not a dep/upstream reason):

```
diff --git a/devel/build b/devel/build
index 2c4f866..eba094e 100755
--- a/devel/build
+++ b/devel/build
@@ -59,7 +59,8 @@ lock() {
             | .build.string = $rendered[0].build.string + "_locked"
             | .requirements.run = (
                   $rendered[0].requirements.run
-                | map(split(" ") | "\(.[0]) ==\(.[1]) \(.[2])"))
+              | map(select(startswith("__glibc") | not))
+              | map(split(" ") | "\(.[0]) ==\(.[1]) \(.[2])"))
         ' \
         < src/recipe.yaml \
         > locked/recipe.yaml
diff --git a/src/recipe.yaml b/src/recipe.yaml
index 79ce175..1b72919 100644
--- a/src/recipe.yaml
+++ b/src/recipe.yaml
@@ -98,4 +98,4 @@ requirements:
     #
     # ¹ <https://github.com/conda-forge/c-ares-feedstock/pull/38>
     # ² <https://github.com/nextstrain/conda-base/issues/92>
-    - c-ares <1.33.0
+    - sel(linux): __glibc <2.28
```

**Pull request merged**